### ISSUE TITLE
Bug: Submit screen - user can go back with keyboard navigation

### ISSUE THREAD

**Issue by EkaterinaVu** [ditdot-dev/vue-flow-form#118]
**Describe the bug**
On the final, submit screen user can go back with keyboard navigation although it should be disabled after submit.
